test/e2e/framework/persistentvolumeclaim.go

```diff
@@ -0,0 +1,83 @@
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/klog/v2"
+)
+
+// CreatePVC create PersistentVolumeClaim.
+func CreatePVC(client kubernetes.Interface, pvc *corev1.PersistentVolumeClaim) {
+	ginkgo.By(fmt.Sprintf("Creating PersistentVolumeClaim(%s/%s)", pvc.Namespace, pvc.Name), func() {
+		_, err := client.CoreV1().PersistentVolumeClaims(pvc.Namespace).Create(context.TODO(), pvc, metav1.CreateOptions{})
+		gomega.Expect(err).ShouldNot(gomega.HaveOccurred())
+	})
+}
+
+// RemovePVC delete PersistentVolumeClaim.
+func RemovePVC(client kubernetes.Interface, namespace, name string) {
+	ginkgo.By(fmt.Sprintf("Removing PersistentVolumeClaim(%s/%s)", namespace, name), func() {
+		err := client.CoreV1().PersistentVolumeClaims(namespace).Delete(context.TODO(), name, metav1.DeleteOptions{})
+		gomega.Expect(err).ShouldNot(gomega.HaveOccurred())
+	})
+}
+
+// WaitPVCPresentOnClustersFitWith wait PersistentVolumeClaim present on clusters sync with fit func.
+func WaitPVCPresentOnClustersFitWith(clusters []string, namespace, name string, fit func(pvc *corev1.PersistentVolumeClaim) bool) {
+	ginkgo.By(fmt.Sprintf("Waiting for pvc(%s/%s) synced on member clusters", namespace, name), func() {
+		for _, clusterName := range clusters {
+			WaitPVCPresentOnClusterFitWith(clusterName, namespace, name, fit)
+		}
+	})
+}
+
+// WaitPVCPresentOnClusterFitWith wait pvc present on member cluster sync with fit func.
+func WaitPVCPresentOnClusterFitWith(cluster, namespace, name string, fit func(pvc *corev1.PersistentVolumeClaim) bool) {
+	clusterClient := GetClusterClient(cluster)
+	gomega.Expect(clusterClient).ShouldNot(gomega.BeNil())
+
+	klog.Infof("Waiting for pvc(%s/%s) synced on cluster(%s)", namespace, name, cluster)
+	gomega.Eventually(func() bool {
+		pvc, err := clusterClient.CoreV1().PersistentVolumeClaims(namespace).Get(context.TODO(), name, metav1.GetOptions{})
+		if err != nil {
+			return false
+		}
+		return fit(pvc)
+	}, pollTimeout, pollInterval).Should(gomega.Equal(true))
+}
+
+// WaitPVCDisappearOnCluster wait PersistentVolumeClaim disappear on cluster until timeout.
+func WaitPVCDisappearOnCluster(cluster, namespace, name string) {
+	clusterClient := GetClusterClient(cluster)
+	gomega.Expect(clusterClient).ShouldNot(gomega.BeNil())
+
+	klog.Infof("Waiting for pvc(%s/%s) disappear on cluster(%s)", namespace, name, cluster)
+	gomega.Eventually(func() bool {
+		_, err := clusterClient.CoreV1().PersistentVolumeClaims(namespace).Get(context.TODO(), name, metav1.GetOptions{})
+		if err == nil {
+			return false
+		}
+		if apierrors.IsNotFound(err) {
+			return true
+		}
+
+		klog.Errorf("Failed to get pvc(%s/%s) on cluster(%s), err: %v", namespace, name, cluster, err)
+		return false
+	}, pollTimeout, pollInterval).Should(gomega.Equal(true))
+}
+
+// WaitPVCDisappearOnClusters wait service disappear on member clusters until timeout.
```

Comment:
```suggestion
// WaitPVCDisappearOnClusters: Wait for the service to disappear on member clusters until timeout.
```

Not sure whether my suggestion makes sense. It looks like a function description to me. Another thing is, why do we say "wait **service** disappear" in the description while the function name is "Wait**PVC**"? Shouldn't we say "Wait for the **PVC** to disappear"?

Reply:
Yes, that's an error, `wait for the PVC to disappear` is right

Reply:
I have to use `WaitPVCDisappearOnClusters` instead of `WaitPVCDisappearOnClusters:`, because use `WaitPVCDisappearOnClusters:` my goland will give me a warning, may this is a convention in `go`